core/bot_commands.py

```diff
@@ -1,4 +1,18 @@
 from nio import RoomMessageText, AsyncClient, MatrixRoom
+import re
+
+
+def _parse_args(input: str) -> "list[str]":
```

Comment:
While it does seem to have a similar effect, at least in PyCharm, please change `"list[str]"` to `List[str]` (`from typing import List`) for consistency with existing code.